hubgrep/frontend_blueprint/routes/search.py

```diff
@@ -44,7 +45,7 @@ def search():
                            search_results=results_paginated,
                            search_feedback=search_feedback,
                            pagination_links=pagination_links,  # [PageLink] namedtuples
-                           external_errors=external_errors)  # TODO these errors should be formatted to text that is useful for a enduser
+                           failed_requests=failed_requests)  # TODO these errors should be formatted to text that is useful for a enduser
```

Comment:
I changed this name as its not a more specific thing, and I think it will always be about errors from failed requests.